shell/config/labels-annotations.js

```diff
@@ -172,3 +172,7 @@ export const SYSTEM_LABELS = [
   'node.kubernetes.io',
   'egress.rke2.io'
 ];
+
+export const CLOUD_CREDENTIALS = [
+  'cattle.io/expiration-date'
```

Comment:
this is now `cattle.io/expiration-timestamp`

Reply:
now `rancher.io/expiration-timestamp`...

Reply:
for reference https://github.com/rancher/rancher/pull/47488/files#diff-10c41bfb5e92c85f5c529be14f327d105ed94394ed74f8b3c2e46b523a506af0R27